package.json

```diff
@@ -111,7 +111,7 @@
     "browserify": "^16.2.3",
     "chalk": "^2.4.1",
     "chrome-devtools-frontend": "1.0.727089",
-    "codecov": "^3.2.0",
+    "codecov": "^3.7.0",
```

Comment:
will need to `yarn add -D codecov@^3.7.0` to get the `yarn.lock` file to update too

Reply:
thanks! added it